scripts/docker_files/docker_file_wheelbuilder_linux_mpi/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM quay.io/pypa/manylinux2010_x86_64
+
+ENV MMG_ROOT=/external_libraries/mmg/mmg_5_4_1
+ENV LD_LIBRARY_PATH ${LD_LIBRARY_PATH}:{MMG_ROOT}/lib
+
+RUN yum install -y git wget zip
+
+RUN yum install -y libgfortran.x86_64 blas-devel lapack-devel
```

Comment:
Why is this needed?

Reply:
Mmm yep that's no longer needed. Copy paste frenzy